### ISSUE TITLE
SNAPSHOT dependency

### ISSUE THREAD

**Issue by cosminci** [nickman/tsdb-csf#6]
The project has a dependency to `heliosutils-1.0-SNAPSHOT`, which also needs to be cloned and built locally. It might be worth mentioning this in the description, unless `helioutils` is already in a public maven repo.

**Comment by nickman** [comment]
Yes, thanks for reporting. I have my sonatype papers but I need to add
public repo publication to the build chain.
